src/JWT/Algorithms/RS256Algorithm.cs

```diff
@@ -37,7 +37,7 @@ public byte[] Sign(byte[] key, byte[] bytesToSign)
 
         private static RSA GetRSA(X509Certificate2 cert)
         {
-#if NETSTANDARD
+#if NETSTANDARD1_3

```

Comment:
The project defines it as just NESTANDARD now, to avoid changing this line every time we try to change the target profile, see https://github.com/paule96/jwt/blob/01dbe5fc88121c6864eb45b0d934512f4dd7540c/src/JWT/JWT.csproj#L12

Reply:
Then this line in the project won't work. Because i get a compile error if I don't change the line....

Reply:
Then this line in the project won't work. Because i get a compile error if I don't change the line....